home-application/src/main/java/com/softserveinc/ita/homeproject/application/api/PollApiImpl.java

```diff
@@ -16,32 +17,39 @@
 import javax.ws.rs.ext.Provider;
 
 import com.softserveinc.ita.homeproject.api.PollsApi;
+import com.softserveinc.ita.homeproject.application.config.HomeUserWrapperDetails;
+import com.softserveinc.ita.homeproject.application.service.impl.HomeUserDetailsService;
+import com.softserveinc.ita.homeproject.homedata.entity.User;
+import com.softserveinc.ita.homeproject.homeservice.dto.CreateVoteDto;
 import com.softserveinc.ita.homeproject.homeservice.dto.HouseDto;
 import com.softserveinc.ita.homeproject.homeservice.dto.PollDto;
 import com.softserveinc.ita.homeproject.homeservice.dto.PollQuestionDto;
 import com.softserveinc.ita.homeproject.homeservice.service.HouseService;
 import com.softserveinc.ita.homeproject.homeservice.service.PollHouseService;
 import com.softserveinc.ita.homeproject.homeservice.service.PollQuestionService;
 import com.softserveinc.ita.homeproject.homeservice.service.PollService;
+import com.softserveinc.ita.homeproject.homeservice.service.VoteService;
 import com.softserveinc.ita.homeproject.model.CreateQuestion;
+import com.softserveinc.ita.homeproject.model.CreateVote;
 import com.softserveinc.ita.homeproject.model.HouseLookup;
 import com.softserveinc.ita.homeproject.model.PollStatus;
 import com.softserveinc.ita.homeproject.model.PollType;
 import com.softserveinc.ita.homeproject.model.QuestionType;
 import com.softserveinc.ita.homeproject.model.ReadHouse;
 import com.softserveinc.ita.homeproject.model.ReadMultipleChoiceQuestion;
 import com.softserveinc.ita.homeproject.model.ReadPoll;
+import com.softserveinc.ita.homeproject.model.ReadQuestion;
+import com.softserveinc.ita.homeproject.model.ReadVote;
 import com.softserveinc.ita.homeproject.model.UpdateQuestion;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.Page;
 import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.stereotype.Component;
 
 @Provider
 @Component
 public class PollApiImpl extends CommonApi implements PollsApi {
-    @Autowired
-    private PollService pollService;
```

Comment:
It is pretty hard to review your changes when you have everything shuffled. Please keep it in the original place in order to have precise diffs.

Reply:
Done. I tried to take away unnecessary changes.